lib/Backend/GlobOpt.cpp

```diff
@@ -9585,6 +9584,32 @@ GlobOpt::TypeSpecializeInlineBuiltInBinary(IR::Instr **pInstr, Value *src1Val, V
             break;
         }
 
+        case Js::OpCode::InlineMathPow:
+        {
+#ifndef _M_ARM
```

Comment:
ARM has its specific MD (it doesn't use LowerMDShared.cpp) specific lowering and requires separated implementation, so doesn't fall into this general path.

---
In reply to: [57507086](https://github.com/Microsoft/ChakraCore/pull/644#discussion_r57507086) [](ancestors = 57507086)